app/(dashboard)/dashboard/_components/ParticipantsTable/ParticipantsTable.tsx

```diff
@@ -1,71 +1,70 @@
+import { useState } from 'react';
+import { DeleteParticipantsDialog } from '~/app/(dashboard)/dashboard/participants/_components/DeleteParticipantsDialog';
 export const ParticipantsTable = ({
   initialData,
 }: {
-  initialData: Participant[];
+  initialData: ParticipantWithInterviews[];
 }) => {
-  const [seletedParticipant, setSeletedParticipant] = useState<string | null>(
-    null,
-  );
-  const [showModal, setShowModal] = useState(false);
-
-  const {
-    isLoading,
-    refetch,
-    data: participants,
-  } = api.participant.get.all.useQuery(undefined, {
-    initialData,
-    refetchOnMount: false,
-    onError(error) {
-      // eslint-disable-next-line no-console
-      console.error(error);
+  const { isLoading, data: participants } = api.participant.get.all.useQuery(
+    undefined,
+    {
+      initialData,
+      refetchOnMount: false,
+      onError(error) {
+        // eslint-disable-next-line no-console
+        console.error(error);
+      },
     },
-  });
-
-  const { mutateAsync: deleteParticipants } =
-    api.participant.delete.byId.useMutation();
-
-  const editParticipant = (identifier: string) => {
-    setSeletedParticipant(identifier);
-    setShowModal(true);
-  };
+  );
+  const [participantsToDelete, setParticipantsToDelete] =
+    useState<ParticipantWithInterviews[]>();
+  const [showDeleteModal, setShowDeleteModal] = useState(false);
 
-  const handleDelete = async (data: Participant[]) => {
-    await deleteParticipants(data.map((d) => d.identifier));
-    await refetch();
+  const handleDelete = (data: ParticipantWithInterviews[]) => {
+    setParticipantsToDelete(data);
+    setShowDeleteModal(true);
   };
 
   return (
     <>
       <div className="flex gap-2">
-        <ParticipantModal
-          open={showModal}
-          setOpen={setShowModal}
-          existingParticipants={participants}
-          editingParticipant={seletedParticipant}
-          setEditingParticipant={setSeletedParticipant}
-        />
+        <AddParticipantButton existingParticipants={participants} />
         <ImportCSVModal />
         <ExportCSVParticipants participants={participants} />
         <DeleteAllParticipantsButton />
       </div>
       {isLoading && <div>Loading...</div>}
+      <DeleteParticipantsDialog
+        open={showDeleteModal}
+        setOpen={setShowDeleteModal}
+        participantsToDelete={participantsToDelete || []}
+      />
       <DataTable
-        columns={ParticipantColumns(editParticipant, handleDelete)}
+        columns={ParticipantColumns()}
         data={participants}
         filterColumnAccessorKey="identifier"
         handleDeleteSelected={handleDelete}
+        actions={[
+          {
+            id: 'actions',
+            header: () => <Settings />,
+            cell: ({ row }) => {
+              return <ActionsDropdown row={row} participants={participants} />;
+            },
+          },
+        ]}
```

Comment:
I think a better pattern for this would be:

```
<DataTable
  actions={ActionsDropdown}
  actionRowHeader={Settings}
/>
```

Although its convenient to pass in the structure exactly as you need to consume it in data table, it doesn't really make sense as a prop. Remember that when we expose props we are thinking of the consumer's interface for the component. The consumer doesn't know or care how the component works internally, and usually is most concerned with being able to quickly understand the props interface. Doing it the way I suggested will give us that, while also giving is simpler typing.

A couple of things to point out with your approach:
  - you're passing participants in to the actions dropdown. I would suggest making it so that whatever component is passed as the actions component is passed the row and the data as properties by the data table, since it is already being given the data prop itself. This simplifies things, and should be able to be typed so that the component you pass must accept those props (though be careful with this - maybe they are both optional?). 
  - its a good idea to separate out props that are optional from ones that are required. For example, `actionRowHeader` can default to something sensible (although I suggest maybe making it blank by default, as I did find the settings cog confusing) and therefore be an optional property. 

Hope its obvious enough how to make this work inside data table, but let me know if not.

Reply:
resolved with [13bdd5a](https://github.com/complexdatacollective/Fresco/pull/25/commits/13bdd5ae29b795c64c7a197bdf40db65c8eabc69)

I made `actionsHeader` blank by default, and opted to leave it as blank for both of the tables for now. If we want to add something back in for that, we can.